reference/features/configuration.md

```diff
@@ -32,342 +30,351 @@ Server is listening on: http://127.0.0.1:7700
 - [Dumps](/reference/features/configuration.md#dumps-destination)
-  - [Dumps Destination](/reference/features/configuration.md#dumps-destination)
-  - [Import Dump](/reference/features/configuration.md#import-dump)
-  - [Dump Batch Size](/reference/features/configuration.md#dump-batch-size)
-- [Max MDB Size](/reference/features/configuration.md#max-mdb-size)
-- [Max UDB Size](/reference/features/configuration.md#max-udb-size)
-- [SSL Configuration](/reference/features/configuration.md#ssl-authentication-path):
-  - [SSL Authentication Path](/reference/features/configuration.md#ssl-authentication-path)
-  - [SSL Certicates Path](/reference/features/configuration.md#ssl-certificates-path)
-  - [SSL Key Path](/reference/features/configuration.md#ssl-key-path)
-  - [SSL OCSP Path](/reference/features/configuration.md#ssl-ocsp-path)
-  - [SSL Require Auth](/reference/features/configuration.md#ssl-require-auth)
-  - [SSL Resumption](/reference/features/configuration.md#ssl-resumption)
-  - [SSL Tickets](/reference/features/configuration.md#ssl-tickets)
-- [Disable Sentry](/reference/features/configuration.md#disable-sentry)
+  - [Dumps destination](/reference/features/configuration.md#dumps-destination)
+  - [Import dump](/reference/features/configuration.md#import-dump)
+  - [Dump batch size](/reference/features/configuration.md#dump-batch-size)
+- [Max MDB size](/reference/features/configuration.md#max-mdb-size)
+- [Max UDB size](/reference/features/configuration.md#max-udb-size)
+- [Payload limit size](/reference/features/configuration.md#payload-limit-size)
+- [Snapshots](/reference/features/configuration.md#schedule-snapshot-creation):
+  - [Schedule snapshot creation](/reference/features/configuration.md#schedule-snapshot-creation)
+  - [Snapshot destination](/reference/features/configuration.md#snapshot-destination)
+  - [Snapshot interval](/reference/features/configuration.md#snapshot-interval)
+  - [Import snapshot](/reference/features/configuration.md#import-snapshot)
+  - [Ignore missing snapshot](/reference/features/configuration.md#ignore-missing-snapshot)
+  - [Ignore snapshot if DB exists](/reference/features/configuration.md#ignore-snapshot-if-db-exists)
+- [SSL configuration](/reference/features/configuration.md#ssl-authentication-path):
+  - [SSL authentication path](/reference/features/configuration.md#ssl-authentication-path)
+  - [SSL certificates path](/reference/features/configuration.md#ssl-certificates-path)
+  - [SSL key path](/reference/features/configuration.md#ssl-key-path)
+  - [SSL OCSP path](/reference/features/configuration.md#ssl-ocsp-path)
+  - [SSL require auth](/reference/features/configuration.md#ssl-require-auth)
+  - [SSL resumption](/reference/features/configuration.md#ssl-resumption)
+  - [SSL tickets](/reference/features/configuration.md#ssl-tickets)
 
 ### Database path
 
 **Environment variable**: `MEILI_DB_PATH`
 **CLI option**: `--db-path`
+**Default value**: `"./data.ms"`
```

Comment:
I think there is a minor inconsistency in our documentation style here. The default value here is `"./data.ms"`, whereas similar options like snapshot and dump directories have a default value without the `.`, e.g. `snapshots/` and `dumps/`.

Supposing that this is a problem w/ our documentation and not related to how MeiliSearch actually interprets the input, I think we should change this one to match the others:

```suggestion
**Default value**: `data.ms/`
```

Can someone from the core team point to the code where this is implemented? @MarinPostma @ManyTheFish